scripts/evolving.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$data = file_get_contents('/var/www/log/time_to_evolve.txt');
+$replace = file_get_contents('/var/www/log/replacement.txt');
+$list = json_decode($data, true);
+
+foreach ($list as $value) {
+    // need to open the file to be modified
+    $file_data = file_get_contents($value['file_name']);

```

Comment:
And here.

Reply:
This is coming from a file that was created in a search so this file had better exist.